packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/InAppPurchasePluginTests.swift

```diff
@@ -82,7 +82,7 @@ final class InAppPurchasePluginTests: XCTestCase {
       }
 
       XCTAssertEqual(unwrappedProducts.count, 1)
-      XCTAssertEqual(response.invalidProductIdentifiers, nil)
+      XCTAssertEqual(response.invalidProductIdentifiers, [])
```

Comment:
why this behavior change?

Reply:
obj-c let me pass in a nil with no objections, but invalidProductIdentifiers is not an optional field in swift version of the api https://developer.apple.com/documentation/storekit/skproductsresponse/1505985-invalidproductidentifiers